ADAL/src/utils/ios/ADKeychainUtil.m

```diff
@@ -53,15 +53,16 @@ + (NSString*)retrieveTeamIDFromKeychain:(ADAuthenticationError * __autoreleasing
     NSDictionary *query = @{ (id)kSecClass : (id)kSecClassGenericPassword,
                              (id)kSecAttrAccount : @"teamIDHint",
                              (id)kSecAttrService : @"",
-                             (id)kSecAttrAccessible : (id)kSecAttrAccessibleAlways,
                              (id)kSecReturnAttributes : @YES };
     CFDictionaryRef result = nil;
     
     OSStatus status = SecItemCopyMatching((__bridge CFDictionaryRef)query, (CFTypeRef *)&result);
     
     if (status == errSecItemNotFound)
     {
-        status = SecItemAdd((__bridge CFDictionaryRef)query, (CFTypeRef *)&result);
+        NSMutableDictionary* addQuery = [query mutableCopy];
+        [addQuery setObject:(id)kSecAttrAccessibleAlways forKey:(id)kSecAttrAccessible];
```

Comment:
This change still creates the item using the kSecAttrAccessibleAlways attribute. However, it removes it from the SecItemCopyMatching query which will allow it to be found regardless of its kSecAttrAccessible value which will be kSecAttrAccessibleAlwaysThisDeviceOnly if it was created by an Intune MAM managed application.